### PR TITLE
onFocus and accessibility support

### PR DESCRIPTION
**Pull request by feldmisco**
Added onFocus property for onFocus handling, and some accessibility properties and traits for the search bar buttons

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -213,7 +219,11 @@ export default class Search extends Component {
           >
           {
             !hideBack &&
-            <TouchableOpacity onPress={onBack || this.hide}>
+            <TouchableOpacity
+              accessible={true}
+              accessibilityComponentType="button"
+              accessibilityLabel={backButtonAccessibilityLabel ? backButtonAccessibilityLabel : "Navigate Up"}
```
Comment (daveols) [inline index.js]: Instead of the ternary statement here, just add "Navigate Up" as a default prop.

File: index.js [hunk 2]
```diff
@@ -249,7 +260,11 @@ export default class Search extends Component {
               autoCorrect={autoCorrect}
               autoCapitalize={autoCapitalize}
             />
-            <TouchableOpacity onPress={hideX || this.state.input === '' ? null : this._clearInput}>
+            <TouchableOpacity
+              accessible={true}
+              accessibilityComponentType="button"
+              accessibilityLabel={closeButtonAccessibilityLabel ? closeButtonAccessibilityLabel : "Clear Search Text"}
```
Comment (daveols) [inline index.js]: Same thing here.

File: index.js (hunk 1 above)
Comment (feldmisco) [inline index.js]: Thank you for the feedback! I'll work on updating the request shortly.

**Comment by feldmisco** [comment]
Okay, I have updated the ternary statements to be default props instead.

**Comment by daveols** [comment]
👍 

Just update the readme and I'll merge it! Thanks

**Comment by feldmisco** [comment]
Updated the README.md 👍 Thank you!

**Pull request merged**